appsec-kit-v24/pom.xml

```diff
@@ -32,6 +32,21 @@
             <version>${vaadin24.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin</artifactId>
+            <version>${vaadin24.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-html-components</artifactId>
+            <version>24.1.10</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jsoup</groupId>
+            <artifactId>jsoup</artifactId>
+            <version>1.16.1</version>
+        </dependency>
```

Comment:
Where is this dependency needed? I would expect to be transitive from Flow.

Reply:
It's a leftover from some early logic. Removed it.